### ISSUE TITLE
Adding ticks, min and max value on the gauge

### ISSUE THREAD

**Issue by YacineGameDev** [ashish-chopra/ngx-gauge#27]
it would be very nice if it was possible to display the min and max value and also ticks ( let's say every 10 km/h for example)

**Comment by fernandocode** [comment]
@Observablerxjs, Can you give us an example of what you would like to do? I could not understand in his explanation, could it represent the expected result visually?

**Comment by YacineGameDev** [comment]
![3z5uo](https://user-images.githubusercontent.com/23528812/40626014-e7a46d86-6283-11e8-8e93-f27afdec9c80.png)
Something like this, with ticks and the min and max value displayed

**Comment by ashish-chopra** [comment]
Ticks could be an acceptable feature, but putting a needle is not acceptable. Because these types of gauges already exist on the Internet. I do not want to recreate it. Tick or balloons (like Angular Material) on hover is a usability enhancement which I can think of.

The work the needle performs on the given snapshot is done by an analog circular arc with a forgroundColor growing/shrinking on the gauge.

**Comment by YacineGameDev** [comment]
I totally agree with you, it was just an example to show the ticks and the ballons. Those two would give a big improvement to the gauge.

**Comment by neilmaynard** [comment]
Hi! Any updates on adding ticks to the gauge anyone?

**Comment by enlight3d** [comment]
Hello, any plans on adding the ticks ? I've searched for another library in the meantine and I can't find one that looks as clean as this one...

**Comment by ashish-chopra** [comment]
Closing — this is covered by the `markers` input (added some releases back).
You can render ticks, custom labels per tick, and min/max markers all
through the same API:

```ts
markers = {
  '0':   { type: 'line', color: '#555', size: 8, label: '0' },
  '25':  { type: 'line', color: '#555', size: 6 },
  '50':  { type: 'line', color: '#555', size: 8, label: '50' },
  '75':  { type: 'line', color: '#555', size: 6 },
  '100': { type: 'line', color: '#555', size: 8, label: '100' },
};
```
A needle/pointer remains out of scope for this component (see my earlier
comment on this thread). If anyone wants a showMinMax convenience shortcut
on top of markers, please open a focused feature request. Thanks!
